pages/blog/index.vue

```diff
@@ -97,18 +97,31 @@ export default {
     const resolve = require.context('~/content/blog/', true, /\.md$/)
     const imports = resolve.keys().map((key) => resolve(key))
 
+    const tags = {
+      tokens: 'Tokens',
+    }
+
     const latestPosts = imports.sort(
       (a, b) => +new Date(b.attributes.date) - +new Date(a.attributes.date)
     )
 
-    const tokensPosts = latestPosts.filter(
-      (post) => post.attributes.tags === 'Tokens'
+    return latestPosts.reduce(
```

Comment:
hhmm, I don't think we need reduce here

just use `shift()` here to remove first array
https://github.com/kodadot/nft-gallery/blob/d09c3b68c076f7214599f1f2de8f32d12f2236c9/pages/blog/index.vue#L70

add more filter here to exclude first array from `latestPosts`
https://github.com/kodadot/nft-gallery/blob/d09c3b68c076f7214599f1f2de8f32d12f2236c9/pages/blog/index.vue#L104-L106

Reply:
oh nvm, this is ok 👍